exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/aggregate/HashAggregator.java

```diff
@@ -47,10 +47,7 @@
   // OK - batch returned, NONE - end of data, RESTART - call again
   public enum AggIterOutcome { AGG_OK, AGG_NONE, AGG_RESTART }
 
-  public abstract void setup(HashAggregate hashAggrConfig, HashTableConfig htConfig, FragmentContext context,
-                             OperatorStats stats, OperatorContext oContext, RecordBatch incoming, HashAggBatch outgoing,
-                             LogicalExpression[] valueExprs, List<TypedFieldId> valueFieldIds, TypedFieldId[] keyFieldIds,
-                             VectorContainer outContainer) throws SchemaChangeException, IOException, ClassTransformationException;
+  public abstract void setup(HashAggregate hashAggrConfig, HashTableConfig htConfig, FragmentContext context, OperatorStats stats, OperatorContext oContext, RecordBatch incoming, HashAggBatch outgoing, LogicalExpression[] valueExprs, List<TypedFieldId> valueFieldIds, TypedFieldId[] keyFieldIds, VectorContainer outContainer, int extraRowBytes) throws SchemaChangeException, IOException, ClassTransformationException;
```

Comment:
Not sure that putting all items on one big line is an improvement over the arg-per-line format previously.

Also, see note above: a large number of arguments suggest a muddy design with one class trying to do far too much.

Reply:
That was one of the IDE's ideas ....
And simplification could be done as part of future cleanup work (like DRILL-5779)